http/templates/bracket_detail.html.tera

```diff
@@ -11,7 +11,9 @@
 
     {% for round in bracket.rounds %}
     <div class="subsection-container">
+        {% if not bracket.is_round_robin %}
```

Comment:
this is mildly gross and it's possible the True Solution is to have separate `swiss_bracket_detail.html.tera` and `rr_bracket_detail.html.tera` templates to maximize beauty